CWE/localisation/Chrisfox's events.csv

```diff
@@ -461,4 +461,8 @@ hypersonic_glide_vehicle;HGV;;;;;;;;;;;;;x;;;;;;;;;;;;;;;;
 EVTDESC8022960;The United Democratic Forces, still reeling from its recent defeat in the country's elections, has seemingly gained the backing of the United States in its fight to overturn its loss and demand a resignation of the current government. Already funding, technical aid, and help with coordination has allowed a rapid mobilization of forces across the nation - with the US itself ramping up pressure on Bulgaria and Mladenov's administration, threatening to exclude the nation from financial endeavors should they defy the demands of the opposition and US diplomats.;;;;;;;;;;X;;;;;;;
-EVTDESC8022961;The BSP, still in chaos from the resignation of many of its leadership who had helped in the nation's transition to a more western-oriented regime, has continued to face backlash and calls for a complete reversal of the 1990 elections by opposition forces. While counter-protests have risen in opposition to the demands of the opposition, the UDF with its funding and foreign aid has managed to overshadow them - growing its support with promises of US aid to the nation and fears of a return to the communist regime of only a few years ago. \n\n Already many with the BSP have prepared to completely concede to the opposition, divisions and rifts forming as pressure from within and without increasingly points towards fresh elections and the introduction of yet more reforms in an attempt to hold together a unified and stable Bulgaria.;;;;;;;;;;X;;;;;;;
\ No newline at end of file
+EVTDESC8022961;The BSP, still in chaos from the resignation of many of its leadership who had helped in the nation's transition to a more western-oriented regime, has continued to face backlash and calls for a complete reversal of the 1990 elections by opposition forces. While counter-protests have risen in opposition to the demands of the opposition, the UDF with its funding and foreign aid has managed to overshadow them - growing its support with promises of US aid to the nation and fears of a return to the communist regime of only a few years ago. \n\n Already many with the BSP have prepared to completely concede to the opposition, divisions and rifts forming as pressure from within and without increasingly points towards fresh elections and the introduction of yet more reforms in an attempt to hold together a unified and stable Bulgaria.;;;;;;;;;;X;;;;;;;
+WEC;Western Canada;;x;;;;;;;;;;;;;;;;
+WEC_ADJ;Western Canadian;;x;;;;;;;;;;;;;;;;
+american_spanish;Hispano;;x;;;;;;;;;;;;;;;;
+metis;Metis;;x;;;;;;;;;;;;;;;;
```

Comment:
Is there an entry for XIJ?

Reply:
Sorry, I am not good at this. It seems to have overwritten my changes for some reason and included things from commits made after this one (which is exceptionally annoying). 

I had originally uploaded JUST the pop changes, the bug fix for the OGAS event, the party changes, and the revolter. Afterwards I went back to my fork and added in the commit. It seems to have added things from the commits I put in after to this pull request without actually including everything.

Reply:
Alright